compiler/src/org/jetbrains/dukat/compiler/translator/JSModuleTranslator.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.dukat.compiler.translator
+
+import org.jetbrains.dukat.translator.InputTranslator
+import org.jetbrains.dukat.astModel.*
```

Comment:
Please, check settings here https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-

We have an agreement not to use star imports. May be we should check how to fail on having them.